### PR TITLE
system-variables: add a warning about tidb_enable_dist_task in v7.1

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a warning about variable `tidb_enable_dist_task` in v7.1.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/tidb/issues/50890

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 1]
```diff
@@ -1196,7 +1196,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 > **警告：**
 >
-> 该功能目前为实验特性，不建议在生产环境中使用。
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能开启时，如果你打算从 v7.1.x 升级到 v7.5.0 或者更高的版本，请确保升级前没有正在运行的 `ADD INDEX` 任务。
```
Comment (Oreoxmt) [inline system-variables.md:1199]:
```suggestion
> 该功能目前为实验特性，不建议在生产环境中使用。在开启 TiDB 后端任务分布式框架时，若计划将 TiDB 从 v7.1.x 升级到 v7.5.0 或更高版本，请确保升级前没有正在运行的 `ADD INDEX` 任务。
```

Comment (Oreoxmt) [reply system-variables.md:1199]: 建议将升级的注意事项放到 tidb-distributed-execution-framework.md 功能文档

https://github.com/pingcap/docs-cn/blob/7a99f3a80143c49e5196972d6c0b6d6da29329f2/tidb-distributed-execution-framework.md?plain=1#L8-L18

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**